examples/getstarted/src/api/address/middlewares/address.js

```diff
@@ -0,0 +1,3 @@
+module.exports = options => {
+  return (ctx, next) => next();
```

Comment:
Is it automatically loaded (since it's directly in a specific API directory)? If it isn't, then where is it used?

Reply:
it's not no and that's juste an example. Either I can remove it or add it to the config

Reply:
I think it would make sense to add it to the config. IMO declaring code that isn't used anywhere can be a bit confusing when a newcomer tries to understand the codebase :slightly_smiling_face: